migrations/1736068540316_create_todos_table.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('todos', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
```

Comment:
Consider adding 'allowNull: false' to the 'id' field to ensure data integrity and prevent null values from being inserted into the primary key column.